vaadin-confirm-dialog-flow-parent/vaadin-confirm-dialog-flow-integration-tests/src/main/java/com/vaadin/flow/component/confirmdialog/tests/FeaturesDiy.java

```diff
@@ -63,24 +61,23 @@ protected ConfirmDialog createConfirmPublishDialog(String sample) {
         ConfirmDialog dialog = new ConfirmDialog();
         dialog.setId(sample);
         dialog.setHeader("Ready to publish?");
-        dialog.setText(
-                new Label("Do you want to publish this post?").getElement());
+        dialog.setText(new Label("Do you want to publish this post?"));
```

Comment:
```suggestion
        dialog.setText(new Paragraph("Do you want to publish this post?"));
```

Stop abusing the label ;)

Reply:
*It wasn't me it was the other guy* 🤪